app/assets/javascripts/components/jquery_wrappers/InputTagit.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class InputTagit extends React.Component {
```

Comment:
What do you think about renaming it to `TaggedInput`? This way the name will be decoupled from `tagit` lib.